letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -458,7 +471,10 @@ def _create_vhost(self, path):
                 is_ssl = True
 
         filename = get_file_path(path)
-        is_enabled = self.is_site_enabled(filename)
+        if self.conf("handle-sites"):
+            is_enabled = self.is_site_enabled(filename)
+        else:
+            is_enabled = True
```

Comment:
Outside of debian based distros there's no concept of sites-available/sites-enabled, everything that's found in the path is parsed (and hence enabled). 

If a2ensite is available in OS, handle-sites shouls be set true on constants for it, and vhost-root pointed to a directory where a2ensite picks it up. If your platform doesn't support enabling, handle-sites should be set false and vhost-root pointed to a directory that's parsed by apache.